src/components/KeyAndTuningButton.tsx

```diff
@@ -1,19 +1,40 @@
-import { useState } from 'react';
-import Stack from '@mui/material/Stack';
+import React, { useState } from 'react';
 import Button from '@mui/material/Button';
+import { Box } from '@mui/material';
 
-export default function BasicButtons() {
+interface RootOption {
+  value: string;
+  label: string;
+}
+
+interface KeyAndTuningButtonProps {
+  isIncomplete: boolean;
+  selectedRoot: RootOption | null;
+  selectedMode: string | null;
+  selectedTuning: string | null;
+}
+
+const KeyAndTuningButton: React.FC<KeyAndTuningButtonProps> = ({
+  isIncomplete,

```

Comment:
@stenzeladam what is the purpose of including `selectedRoot`, `selectedMode`, `selectedTuning`?